### PR TITLE
Added new symptom and other fix

### PR DESCRIPTION
**Pull request by nikk735**
Fixed some garbage spiked code stuff and added auto brewery symptom that is now much less rng based

good for game, viro needs more random shit, maybe auto brew is a good random medical condition kinda like how appendicitis is treated but someone else can decide that. 

## Changelog
:cl:
add auto brew
fix spiked
/:cl:

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	stealth = -2
+	resistance = 4
+	stage_speed = -5
+	transmittable = -2
+	level = 5 // feel free to change
+	symptom_delay_min = 1
+	symptom_delay_max = 5
+	severity = 3
+	base_message_chance = 5
+	var/Power = 2
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/brew.dm:26]: diseases have a power var at default, which is,  at base, 1

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	name = "Auto-brewery syndrome"
+	desc = "The virus causes the host to produce intoxicating quantities of ethanol in their digestive system."
+	stealth = -2
+	resistance = 4
+	stage_speed = -5
+	transmittable = -2
+	level = 5 // feel free to change
+	symptom_delay_min = 1
+	symptom_delay_max = 5
+	severity = 3
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/brew.dm:24]: bring base severity down to 2

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	base_message_chance = 5
+	var/Power = 2
+	var/dangerous = FALSE
+	threshold_desc = "<b>Transmission 6:</b> Alcohol is produced faster.<br>\
+					  <b>Resistance 6: Alcohol is much more dangerous and can be lethal</b> ."
+
+/datum/symptom/brew/Start(datum/disease/advance/A)
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/brew.dm:31]: add a severitystart proc for the dynamic severity refactor, with +1 severity for each threshold

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+	if(!..())
+		return
+	if(A.properties["transmittable"] >= 6) //faster rate
+		Power = 2.5
+	if(A.properties["resistance"] >= 6) //dangerous alc
+		dangerous = TRUE
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/brew.dm:37]: check Fever code and how the proc is done: the Power var measures intensity, and the dangerous var allows it to go lethal.

File: code/datums/diseases/advance/symptoms/brew.dm [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+/datum/symptom/brew/Activate(var/datum/disease/advance/A)
+	if(!..())
+		return
+
+	if(prob(6.66*Power*((A.stage/10)+1)))
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/brew.dm:42]: i'd totally change this up honestly. As i said before, make it not synthesize alcohol above a certain threshold
(both Volume and Alcohol power influence drunkness and alcohol damage. drunkenness per second is calculated this way: square root of volume * booze_power * [0.005], and liver damage per tick is calculated this way: (square root of volume * (boozepwr ** [1.6]) * [0.005])/150. I'd suggest making the disease ONLY modify drunkness unless lethal, and cap it similarly.

File: code/datums/diseases/advance/symptoms/spiked.dm [hunk 6]
```diff
@@ -49,9 +48,6 @@ Thresholds
 		Power = 2
 
 /datum/symptom/spiked/Activate(var/datum/disease/advance/A)
-	var/mob/living/M = A.affected_mob
```
Comment (zeskorion) [inline code/datums/diseases/advance/symptoms/spiked.dm:52]: why are you removing these

Comment (nikk735) [reply code/datums/diseases/advance/symptoms/spiked.dm:52]: because A.affected_mob is already declared so not point in writing it again?

**Comment by nikk735** [comment]
im gonna leave it as it is because it works well and has no real issues anyone can feel free to change it tho but i like how it is

File: code/datums/diseases/advance/symptoms/spiked.dm (hunk 6 above)
Comment (zeskorion) [reply code/datums/diseases/advance/symptoms/spiked.dm:52]: ...this line makes A.affected_mob M. shorthand to make shit easier, there's a reason almost eveyrthing uses this shit

**Comment by zeskorion** [comment]
> im gonna leave it as it is because it works well and has no real issues anyone can feel free to change it tho but i like how it is

as it stands, i'm not gonna put this up to vote, it's simply not in a suitable state

**Comment by ike709** [comment]
> im gonna leave it as it is because it works well and has no real issues anyone can feel free to change it tho but i like how it is

Yeah that's not how it works.

**Pull request closed without merging**